showcase/app/templates/page-components/tabs.hbs

```diff
@@ -365,9 +365,9 @@
     <button type="button" {{on "click" this.updateBadgesValuesDemo3}}>update badge values</button>
   </div>
   <Hds::Tabs as |T|>
-    <T.Tab @count={{this.badge1_demo3}}>One</T.Tab>
-    <T.Tab @count={{this.badge2_demo3}}>Two</T.Tab>
-    <T.Tab @count={{this.badge3_demo3}}>Three</T.Tab>
+    <T.Tab @count="{{this.badge1_demo3}}">One</T.Tab>
```

Comment:
well you shouldn't be doing this, you should update upstream types to allow numbers instead of hackishly type casting